### DISCUSSION TITLE
crazyflie2-nrf-firmware

### DISCUSSION THREAD

**Discussion by AbhishekCMU** [bitcraze/discussions#300] [Q&A]
Hi,

I am working to replicate the pm_radio of crazyflie 2.1 and used crazyflie2-nrf-firmware repository code to flash my developed board. My PCB schematic is the same as in the radio board schematic of crazyflie 2.1 based on nrf51822. 

Ref link: https://www.bitcraze.io/documentation/hardware/crazyflie_2_1/crazyflie_2.1_schematics_rev.b.pdf

I am trying to test the working of the radio board but didn't find any test procedure document in the repository. Is it possible to guide me on 'how to test the radio board (pm_radio with crazyflie2-nrf-firmware) ' or a testing procedure to follow?

I am looking for communication between two radio boards.

**Comment by knmcguire** [comment]
Hi!

This a bit out of our scope and we are not sure if we can help you with this. Have you tried contacting[ Nordic Semi](https://www.nordicsemi.com/), the manufacturer/developer of the NRF chip themselves?
